MangaManager/ExternalSources/MetadataSources/metadata/AniList.py

```diff
@@ -77,12 +77,12 @@ def trim(value):
         return ret
 
     @classmethod
-    def get_cinfo(cls, series_name) -> ComicInfo | None:
+    def get_cinfo(cls, partial_comic_info) -> ComicInfo | None:
```

Comment:
`partial_comic_info` reminds me of ComicInfo class. 
For readability maybe rename it to something else and add type.

Reply:
That is what it is. It is an instance of ComicInfo to represent what's currently in the UI. 

Because if the scraper needs the year and series and it's filled out, then it can have it. It's partial because it's not fully filled.